src/applications/mhv-landing-page/tests/containers/AppConfig.unit.spec.jsx

```diff
@@ -1,80 +1,78 @@
 import React from 'react';
-import { datadogRum } from '@datadog/browser-rum';
 import { waitFor } from '@testing-library/react';
 import { renderWithStoreAndRouter } from '@department-of-veterans-affairs/platform-testing/react-testing-library-helpers';
+import {
+  setDatadogRumUser,
+  useDatadogRum,
+} from '@department-of-veterans-affairs/mhv/exports';
 import sinon from 'sinon';
 import { expect } from 'chai';
 
 import AppConfig from '../../containers/AppConfig';
 import { appName } from '../../manifest.json';
 
-const initialStateFn = ({ accountUuid = 'test-id' } = {}) => ({
-  user: {
-    profile: {
-      accountUuid,
-    },
-  },
-});
-
 describe(`${appName} -- <AppConfig />`, () => {
-  it('initializes datadog RUM', () => {
-    const useRumSpy = sinon.spy(datadogRum, 'init');
+  const initialStateFn = ({ accountUuid = 'test-id' } = {}) => ({
+    user: {
+      profile: {
+        accountUuid,
+      },
+    },
+  });
 
+  it('initializes datadog RUM', async () => {
```

Comment:
Note: None of these tests worked correctly. All these tests were really failing.
1. DD does not initialize when running on the local environment. This includes both the set and init calls.
2. The `waitFor` was missing the `await` which ignores the result of the expect. This means that any result passes the test. In all these cases, the expect was always a false. 
This change also gets rid of the `Warning: You seem to have overlapping act() calls` warning.